proxy/proxy.go

```diff
@@ -236,6 +236,8 @@ type UpstreamProxy struct {
 
 // ServeHTTP handles the second (reverse-proxying) leg of pomerium's request flow
 func (u *UpstreamProxy) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	_, span := telemetry.StartSpan(r.Context(), fmt.Sprintf("proxy.%s%s", r.Host, r.URL.Path))
+	defer span.End()
 	u.handler.ServeHTTP(w, r)
```

Comment:
Agree, but going to remove per https://github.com/pomerium/pomerium/pull/230#discussion_r306113269

Reply:
Clarification of that comment - I was more wondering if we should remove the 'path' component.  The answer might be "no", though.

Reply:
I guess we have a few options.:

1. Don't track it at all (because it will vary wildly depending on the downstream app being served, and that's out of scope for our app?)
2. Track path and hostname (great because we kinda get back-door tracing even if org doesn't have it in their apps, but bad because now we have a ton of potential traces.
3. Track just hostname not the path (easier to manage UI wise, get a sense for aggregate response times per downstream app, but we aren't very helpful if that app doesn't itself have tracing).

I feel like if we _are_ going to trace downstream routes, we might as well do it on a per-path basis.